day_two_guide/topics/proc_restoring-master.adoc

```diff
@@ -68,13 +68,14 @@ $ diff /tmp/current_packages.txt ${MYBACKUPDIR}/packages.txt
 package lists.
 
 . Restore a system certificate by copying the certificate to the
-`/etc/pki/ca-trust/source/anchors/` directory and execute the `update-ca-trust`:
+`/etc/pki/ca-trust/source/anchors/` directory and execute `update-ca-trust`:
 +
 ----
 $ MYBACKUPDIR=*/backup/$(hostname)/$(date +%Y%m%d)*
-$ sudo cp ${MYBACKUPDIR}/external_certificates/my_company.crt /etc/pki/ca-trust/source/anchors/
+$ sudo cp ${MYBACKUPDIR}/etc/pki/ca-trust/source/anchors/<certificate> /etc/pki/ca-trust/source/anchors/ <1>
```

Comment:
@xingxingxia Can you please review this update? It is to remove the reference to "my_company.crt", which you pointed out was not referenced in the backup document. Thanks!

Reply:
@bergerhoffer this place and below place lgtm, thank you.

Reply:
Thank you!